janusgraph-es/src/main/java/org/janusgraph/diskstorage/es/rest/RestElasticSearchClient.java

```diff
@@ -287,7 +287,7 @@ public RestSearchResponse search(String scrollId) throws IOException {
              path = new StringBuilder(REQUEST_SEPARATOR).append("/_search/scroll?scroll=").append(scrollKeepAlive).toString();
```

Comment:
The one line fixed in this PR looks fine.

Scanning through the rest of the file, there is inconsistent use of `"/"` vs `REQUEST_SEPARATOR`. For example, this path appears to start with `//`. It would be great if you could try to clean that up a bit with this PR.